index.html

```diff
@@ -22,8 +22,15 @@
   <script src="./dist/js/chocolatechip.js"></script>
   <script>
     ChocolateChip.eat({
+      language: 'nl_NL',
+      translations: {
+        // Example how to overwrite text
+        // nl_NL: {
+        //   'bannerTemplate.text': ' Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+        // },
+      },
       includes: ['userPreferences', 'analytics', 'advertisements'],
-      privacyPolicyURL: 'http://nu.nl'
+      privacyPolicyURL: 'http://nu.nl',
```

Comment:
Please don't use an ending comma after the last key in the object while using non-transpiled JavaScript. This makes a lot of sense when using a transpiler like Babel, but is not supported while using non-transpiled browser-code.